platform/windows/os_windows.cpp

```diff
@@ -52,11 +52,13 @@
 #include <direct.h>
 #include <knownfolders.h>
 #include <process.h>
+#include <processenv.h>
 #include <psapi.h>
 #include <regstr.h>
 #include <shlobj.h>
 #include <wbemcli.h>
 #include <wincrypt.h>
+#include <winnt.h>
```

Comment:
Why were these two includes added? They're not needed for the change, were they added automatically by your editor? If they're not needed please remove them